src/ContextTypes/Person.php

```diff
@@ -10,6 +10,11 @@ class Person extends AbstractContext
      * @var array
      */
     protected $structure = [
-        'name' => null,
+        'givenName' => null,
+        'familyName' => null,
+        'email' => null,
+        'image' => null,
```

Comment:
maybe 'ImageObject::class' ?

Reply:
Per the JSON LD spec this is a string, not an object